charts/base/values.yaml

```diff
@@ -148,3 +148,8 @@ deployment: {}
 
 env: dev
 product: application
+
+pdb:
+  enabled: false
+  minAvailable: 1
+  pdbName:
```

Comment:
Is it fine to leave this empty?
Do we want to leave this disabled by default?

Reply:
yes, empty value is ok to have, no values set(or empty) for pdbName field pdb resource will take deploy name,
I think having this new option/ability disabled by default is better, wherever we need it can be enabled